src/wrapper/jnienv.rs

```diff
@@ -968,8 +971,8 @@ impl<'a> JNIEnv<'a> {
     #[allow(unused_unsafe)]
     pub unsafe fn release_string_utf_chars(&self, obj: JString, arr: *const c_char) -> Result<()> {
         non_null!(obj, "release_string_utf_chars obj argument");
+        // todo: use sth like 'jni_exception_safe_call'?
```

Comment:
Or just add a comment?

Reply:
I think a comment would suit better here.

Reply:
Now that I think of it, it might be better to use a separate macro for all such methods to express the intent clearly.